accPlot.py

```diff
@@ -195,7 +195,7 @@ def plotTimeSeries(
             datetime.combine(day + timedelta(days=1), time(0, 0, 0, 0))))
         ax.set_xticks(pd.date_range(start=datetime.combine(day,time(0, 0, 0, 0)),
             end=datetime.combine(day + timedelta(days=1), time(0, 0, 0, 0)),
-            freq='4H'))
+            freq='4H')[:-1])
```

Comment:
The problem I have with this quick fix is that I'm not sure we want to "fix" the issue because it seems to be a problem with matplotlib rather than our code. The same issue was posted here: https://stackoverflow.com/questions/62953704/valueerror-the-number-of-fixedlocator-locations-5-usually-from-a-call-to-set Top comment:

> Let me guess. You used matplotlib version 3.2.2 and today created a new environment with matplotlib 3.3.0. Something is broken in new version although I didn't find an exact bug report yet. I suggest to revert back to 3.2.2 until this is resolved.

I agree with this solution. The code worked for _all_ previous matplotlib versions and stopped working only for the latest 3.3.* 

It's not clear why the bug happens and whether your quick fix will break for people using previous matploblib versions.
I suggest `conda install matplotlib=3.2.2` and be clear about package dependencies of our software (I think this is standard in other projects). As always, feel free to disagree!

Reply:
Thanks for the suggestion. It also works with 3.2.2 :D 

But again, fixing the package dependencies is not what this PR is for. That is a separate issue https://github.com/activityMonitoring/biobankAccelerometerAnalysis/issues/92 and is still outstanding : (

Reply:
Yes, it works for 3.2.2 fortunately, but I haven't tried for even older versions. I think we should just wait until matplotlib fixes this bug.

Reply:
It is going to be a bit tricky because this is this DREAMS project that will need to have the plotting for the trial lolol. Maybe I will just ask them to use my fork instead. There will be too many changes and new things coming and I don't think we will be able to finish all of that in time for PR. Whilst this is not ideal, that's the best that we can do right now.

Reply:
Yes I agree. If your collaborators are adamant of using matplotlib=3.3.* we can point them to your branch.

Reply:
On a related note, I suspect this is more of a change of behaviour than a bug TBH. At least, till today, we are not seeing any bug report yet on Matplotlib repo. Previously, the fixlocator can have different lengths from the axises, now it cannot within 3.3.3. It is probably the `right` behaviour from my current understanding.

Yeah the issue is that the research assistants will be less technically sophisticated and are not well-versed in dependency management.

Reply:
Sure let's discuss this during the next meeting and hopefully when the new dependency management tool is being resolved. This PR should settle properly. The longer it waits, the more time it would take to revisit this PR which is not ideal : (

Reply:
> Yeah the issue is that the research assistants will be less technically sophisticated and are not well-versed in dependency management.

Agree, that's why we need to improve our installation process to make sure we are grabbing the right versions. Today is matplotlib but tomorrow might be pandas. Last time was sklearn and joblib. I will look into this too so we can see what's the best way to approach it. Otherwise we shall expect more of these https://github.com/activityMonitoring/biobankAccelerometerAnalysis/issues/20 in the future.